src/components/Header.css

```diff
@@ -0,0 +1,36 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+
+.Header {
+  display: flex;
+
+  /* justify-content: left; */
+  background-color: #fff;
+  height: 70px;
+  line-height: 70px;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 1px solid #e8e8e8;
+}
+
```

Comment:
- [ ] Please add the icon that I on the most right side of the header. 

![tempsnip](https://user-images.githubusercontent.com/64173189/135726871-350e558d-1a9c-4068-86ae-f153899ca865.png)

👍 💯